utils/task-requests.ts

```diff
@@ -42,7 +42,7 @@ const transformTaskRequests = async (taskRequestsList: TaskRequestType[]) => {
   const usersMap = new Map<string, User>(userList.map((data) => [data.id, data]));
   taskRequestsList.forEach((data) => {
     data.users = data.users.map((userData) => {
-      const { username, first_name: firstName, last_name: lastName, picture }: User = usersMap.get(userData.userId);
+      const { username, first_name: firstName, last_name: lastName, picture }: any = usersMap.get(userData.userId);
```

Comment:
Can't we add type here as well instead of specifying `any`?

Reply:
+1

Reply:
Will have change to `User | undefined`, I think it will be the same as any.  And anyways we are destructuring and collecting only string fields and returning them right away.
Let me know what you think.